src/main/java/com/crud/tasks/domain/Task.java

```diff
@@ -22,4 +20,9 @@
     @Column(name = "description")
     private String content;
 
+    public Task(Long id, String title, String content) {
+        this.id = id;
+        this.title = title;
+        this.content = content;
+    }
```

Comment:
to powinnas moc zamienic na `@AllArgsConstructor`
Upewnij sie ze w intelij masz wlaczone procesowanie adnotacji
Build, Execusion, Deployment > Compiler > Annotation Processors > Enable Annotation Processing